hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/operation/GetClusterMetadataOperation.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl.operation;
+
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.core.Cluster;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.impl.ClusterMetadata;
+import com.hazelcast.jet.impl.JetService;
+import com.hazelcast.jet.impl.execution.init.JetInitDataSerializerHook;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+import com.hazelcast.spi.Operation;
+import com.hazelcast.spi.ReadonlyOperation;
+
+public class GetClusterMetadataOperation extends Operation implements
+        IdentifiedDataSerializable,
+        ReadonlyOperation {
```

Comment:
I was looking at `JobSummaryListOperation` and it's marked as `AllowedDuringPassiveState` while this is marked as read-only. What is the difference?

Reply:
From our point of view there is no difference, so I will align them on `ReadonlyOperation`.